lib/src/database/database_client.dart

```diff
@@ -13,6 +13,7 @@ class DatabaseClient {
         database = await MysqlDriver().init();
         database?.connection.reconnectIfMissingConnection();
         break;
+      case 'postgressql':
```

Comment:
This is a typo and should not be promoted. It is better to at least log a warning so that this typo gets fixed by the developer.

Reply:
Fixed Thank you